executor/executor.go

```diff
@@ -183,6 +184,16 @@ func init() {
 	CheckTableFastBucketSize.Store(1024)
 }
 
+// Start the backend components
```

Comment:
I suggest add some comments since we are introducing the "backend components" new concept to this package.

Reply:
And we did lots of things in init() of executor. It seems that we should move those into Start()?

Reply:
Done